### DISCUSSION TITLE
An info about mount park state (not a bug)

### DISCUSSION THREAD

**Discussion by fenriques** [mworion/MountWizzard4#284] [Q&A] [answered]
hi Michael,
I'm writing a custom script that allows the roof and cover to park only when the mount is already parked.
Looking into the 10Micron driver source code it is not clear to me which INDI property I need to check to assess the mount parking state.
Some properties that I was considering were:
- LX200 10micron.TELESCOPE_PARK. But this is monitoring the park command not the park state.
- LX200 10micron.TELESCOPE_TRACK_STATE is off when parked but also when the mount is not tracking.

MW4 always report the correct parking state: Which property do you test?

Ferrante

**Comment by fenriques** [comment]
It came to my mind that probably in MW4 you connect directly to the mount via lan . So that you can query the mount using its command set and get parameters like gstat

**Comment by mworion** [comment, marked as answer]
Hi Ferrante,
You are guessing right. I use a direct communication with the mount via IP (LAN) and use the command protocol.
Michel

**Reply by fenriques** [reply]
hi Michel, 
is there a document online where I can get the command protocol? 
ferrante

**Comment by mworion** [comment]
Moved it to discussions

**Comment by mworion** [comment]
Ferrante,

that‘s quite simple: if you have an actual firmware: open a browser with IP of you mount and you could see the handbook and the command protocol.

If you are already on beta v3 of MW4, go to Mount Tab, Command Sub tab and call the command protocol (which does more or less the same as described above)

Michel

**Comment by fenriques** [comment]
wow, thanks! it was actually pretty easy
![image](https://user-images.githubusercontent.com/38842860/198539001-1820e874-3593-4b1a-90d5-88bbf7a2de7c.png)

**Comment by mworion** [comment]
You're welcome....

**Comment by fenriques** [comment]
btw I was looking into MW4 connection.py and it seems that Gstat (the command I was thinking to use to check the mount status) is not used in MW4. 
I guess you use another command.

**Comment by fenriques** [comment]
like :Ginfo#...

**Comment by mworion** [comment]
Hi Ferrante,

I use Ginfo as it collects more information the same time, but GStat is good as it provides the same info.

Michel
